libs/designsystem/src/lib/components/chart/chart.component.spec.ts

```diff
@@ -65,13 +65,17 @@ describe('ChartComponent', () => {
   });
 
   it('should have dataLabels enabled as default', () => {
-    expect(component.options.plotOptions.pie.dataLabels.enabled).toBe(true);
+    expect(
+      (component.options.plotOptions.pie.dataLabels as PlotSeriesDataLabelsOptions).enabled
+    ).toBe(true);
   });
 
   it('should disable dataLabels when false', () => {
     component.showDataLabels = false;
     component.ngOnInit();
-    expect(component.options.plotOptions.pie.dataLabels.enabled).toBe(false);
+    expect(
+      (component.options.plotOptions.pie.dataLabels as PlotSeriesDataLabelsOptions).enabled
+    ).toBe(false);
```

Comment:
Consider this instead:
```suggestion
    expect(component.options.plotOptions.series.dataLabels).toEqual({ enabled: true });
```